openssl-sys/src/handwritten/x509.rs

```diff
@@ -291,6 +304,17 @@ extern "C" {
     pub fn X509_set_serialNumber(x: *mut X509, sn: *mut ASN1_INTEGER) -> c_int;
     pub fn X509_get_serialNumber(x: *mut X509) -> *mut ASN1_INTEGER;
 }
+cfg_if! {
+    if #[cfg(ossl110)] {
+        extern "C" {
+            pub fn X509_get_X509_PUBKEY(x: *const X509) -> *mut X509_PUBKEY;
+        }
+    } else {
+        pub unsafe fn X509_get_X509_PUBKEY(x: *const X509) -> *mut X509_PUBKEY {
```

Comment:
This macro redefinition will need to live in the top-level x509.rs file so it exists even when building with bindgen.